Dockerfile.self_hosted

```diff
@@ -0,0 +1,12 @@
+
+ARG BASE_IMAGE_HASH
+FROM nangohq/nango:${BASE_IMAGE_HASH}
```

Comment:
There is a bit of a implicit dependency but I guess it is fine since we only ever build images via the github action above.

Reply:
I wouldn't call it implicit if it's the first line but yes it's no longer a standalone image